js/index.js

```diff
@@ -1,42 +1,155 @@
-    "vision-h4":"Vision",
-    "vision-content": "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
-  },
-  "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
-  },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
-  },
+    "nav": {
+
+        "nav-item-1": "Services",
+        "nav-item-2": "Product",
+        "nav-item-3": "Vision",
+        "nav-item-4": "Features",
+        "nav-item-5": "About",
+        "nav-item-6": "Contact",
+        "img-src": "img/logo.png"
+    },
+
+    "cta": {
+
+        "h1": "DOM Is Awesome",
+        "button": "Get Started",
+        "img-src": "img/header-img.png"
+    },
+
+    "main-content": {
+
+        "features-h4": "Features",
+        "features-content": "Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "about-h4": "About",
+        "about-content": "About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "middle-img-src": "img/mid-page-accent.jpg",
+        "services-h4": "Services",
+        "services-content": "Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "product-h4": "Product",
+        "product-content": "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+        "vision-h4": "Vision",
+        "vision-content": "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    },
+
+    "contact": {
+
+        "contact-h4": "Contact",
+        "address": "123 Way 456 Street Somewhere, USA",
+        "phone": "1 (888) 888-8888",
+        "email": "sales@greatidea.io",
+    },
+
+    "footer": {
+
+        "copyright": "Copyright Great Idea! 2018"
+    },
 };
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//content update
+
+//navigation
+let navButtons = document.querySelectorAll("a");
+
```

Comment:
Not necessary, but for repeated things like this, it's usually better to put them in an array.
`navButtons.forEach((navItem, i) => {
     navItem.innerText = siteContent["nav"]['nav-item-${i+1}'];
})`